src/App.tsx

```diff
@@ -1,18 +1,39 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [currentPage, setCurrentPage] = useState<number>(1);
+  const [perPage, setPerPage] = useState<number>(5);
+
+  const startIndex = (currentPage - 1) * perPage;
+  let endIndex = currentPage * perPage;
+
+  if (endIndex > items.length) {
+    endIndex = items.length;
+  }
```

Comment:
Maybe you can replace this code with a ternary operator?

Reply:
Yeah I can, but I saw decision with Math.min and I liked it more)